openstack/dns/v2/recordsets/testing/fixtures.go

```diff
@@ -0,0 +1,168 @@
+package testing
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/openstack/dns/v2/recordsets"
+	th "github.com/gophercloud/gophercloud/testhelper"
+	"github.com/gophercloud/gophercloud/testhelper/client"
+)
+
+// ListOutput is a sample response to a List call.
+const ListOutput = `
+{
+    "recordsets": [
+        {
+            "description": "This is an example record set.",
+            "links": {
+                "self": "https://127.0.0.1:9001/v2/zones/2150b1bf-dee2-4221-9d85-11f7886fb15f/recordsets/f7b10e9b-0cae-4a91-b162-562bc6096648"
+            },
+            "updated_at": null,
+            "records": [
+                "10.1.0.2"
+            ],
+            "ttl": 3600,
+            "id": "f7b10e9b-0cae-4a91-b162-562bc6096648",
+            "name": "example.org.",
+            "project_id": "4335d1f0-f793-11e2-b778-0800200c9a66",
+            "zone_id": "2150b1bf-dee2-4221-9d85-11f7886fb15f",
+            "zone_name": "example.com.",
+            "created_at": "2014-10-24T19:59:44.000000",
+            "version": 1,
+            "type": "A",
+            "status": "PENDING",
+            "action": "CREATE"
+        },
+        {
+            "description": "This is another example record set.",
+            "links": {
+                "self": "https://127.0.0.1:9001/v2/zones/2150b1bf-dee2-4221-9d85-11f7886fb15f/recordsets/f7b10e9b-0cae-4a91-b162-562bc6096648"
+            },
+            "updated_at": "2017-03-04T14:29:07.000000",
+            "records": [
+                "10.1.0.3",
+                "10.1.0.4"
+            ],
+            "ttl": 3600,
+            "id": "7423aeaf-b354-4bd7-8aba-2e831567b478",
+            "name": "foo.example.org.",
+            "project_id": "4335d1f0-f793-11e2-b778-0800200c9a66",
+            "zone_id": "2150b1bf-dee2-4221-9d85-11f7886fb15f",
+            "zone_name": "example.com.",
+            "created_at": "2014-10-24T19:59:44.000000",
+            "version": 1,
+            "type": "A",
+            "status": "PENDING",
+            "action": "CREATE"
+        }
+    ],
+    "links": {
+        "self": "http://127.0.0.1:9001/v2/zones/2150b1bf-dee2-4221-9d85-11f7886fb15f/recordsets?limit=1"
```

Comment:
We really want to be careful about page types. In this case, it does appear to be a `LinkPageBase`, but these must have `next` fields so that Gophercloud knows how to get to the next page. If the service uses a `LinkPageBase` that has a field named something other than `next`, we have to define a [`NextPageURL`](https://github.com/gophercloud/gophercloud/blob/master/pagination/linked.go#L24) method on the page type.